Tests/Integration/MSFT_xFirewall.Integration.Tests.ps1

```diff
@@ -30,9 +30,14 @@ if (($env:PSModulePath).Split(';') -ccontains $pwd.Path)
     $env:PSModulePath = ($env:PSModulePath -split ';' | Where-Object {$_ -ne $pwd.path}) -join ';'
 }
 
+# Preserve and set the execution policy so that the DSC MOF can be created
+$OldExecutionPolicy = Get-ExecutionPolicy
```

Comment:
I'd suggest here instead doing the following.

````powershell
$executionPolicy = Get-ExecutionPolicy
if ($executionPolicy -ne 'Unrestricted')
{
    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Force
    $rollbackExecution = $true
}

#### do everything here ###

Finally {
   if ($rollbackExection)
   {
        Set-ExectuionPolicy $executionPolicy
    }
}
````

This way we're not attempting to change an Execution Policy that doesn't need to be changed.